Cargo.toml

```diff
@@ -71,6 +71,12 @@ version = "^0.11"
 default-features = false
 features = ["multi_thread"]
 
+[features]
```

Comment:
This is ugly but it is the only way to make PyO3 0.22 work with Clippy. I will not bother opening a bug as the latest version already has a fix

Reply:
Yeah I think this is fine as a temporary fix I had to do the same thing on qiskit recently too: https://github.com/Qiskit/qiskit/pull/13645